### PR TITLE
Adding user authentication

### PR DESCRIPTION
**Pull request by halloffame**
Updating feedback form to only show for logged in users
Cleaning up layout file

### CODE REVIEW

File: app/views/layouts/_feedback_form.html.slim [hunk 1]
```diff
@@ -12,19 +12,22 @@
...
-            .form-group
-              label.required Describe the Issue
-              = text_area_tag 'feedback', nil, class: 'form-control', required: true
+          = text_area_tag 'feedback', nil, class: 'form-control', required: true
+
+        - if current_user
+          = hidden_field_tag 'name', current_user.try(:name)
+          = hidden_field_tag 'email', current_user.try(:email)
+
+        - else
```
Comment (mikestephens) [inline app/views/layouts/_feedback_form.html.slim]: Isn't this unnecessary since you have to be signed in to give feedback?

**Comment by rift137** [comment]
Ok 2 separate things. 
* Merging the login true params isn't sending the user to the login page.
* student-support/app/views/shared/_search_banner_with_dates.html.slim:26 @important_date if not present will error out.

**Comment by halloffame** [comment]
Thanks, good catch

**Comment by rift137** [comment]
:+1:

**Pull request merged**